Chapter-1-initial-architecture/Src/Fitnet.ArchitectureTests/Contracts/ContractsModuleArchitectureTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.ArchitectureTests.Contracts;
+
+using System.Reflection;
+using Common.Predicates;
+using NetArchTest.Rules;
+
+public class ContractsModuleArchitectureTests
+{
+    private const string Contracts = "EvolutionaryArchitecture.Fitnet.Contracts";
+    private const string Event = "Event";
+    private readonly Assembly _solution = typeof(Program).Assembly;
+
+    [Fact]
+    internal void Modules_Should_Not_Use_Contract_Module_Internals_Rather_Communicate_Via_Events()
```

Comment:
It is not a contracts architecture test but a test for other modules. For each module, there should be a separate architecture test that checks it. Otherwise there will be a spaghetti quite quickly.

Reply:
Understood!